examples/api/lib/widgets/custom_multi_child_layout/custom_multi_child_layout.0.dart

```diff
@@ -0,0 +1,109 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for CustomMultiChildLayout. This example creates a
```

Comment:
We put these descriptions into the description at the point where it is linked from the API doc comments, so that this description will show up on the API docs website.

For instance, take a look at the existing example's API doc comment:
https://github.com/flutter/flutter/blob/dffddf00a22e77122d67f4503efef16a46c18ec2/packages/flutter/lib/src/widgets/basic.dart#L2206

It results in the example block here, where you can see the description at the top of the blue block.
https://master-api.flutter.dev/flutter/widgets/CustomMultiChildLayout-class.html